### DISCUSSION TITLE
Crafting a Roadmap

### DISCUSSION THREAD

**Discussion by webcrumbs-community** [webcrumbs-community/webcrumbs#117] [General]
Hello WebCrumbs community!

Let's gather around to map out our future. We're crafting our roadmap and your input is vital. 
Join the conversation—every twist, turn, and waypoint starts here!

**Comment by webcrumbs-community** [comment]
I'm thinking of @vbrazo, @aditya4196 and @awaisarshd to give us their comments on our next steps, and invite everyone else interested!

**Comment by awaisarshd** [comment]
@webcrumbs-community thanks for the invite, may I recommend to gather the info for what it has been done so far to review the state we are in. As this will give a better base to start on and draft a plan for future.

**Reply by webcrumbs-community** [reply]
Agree, @awaisarshd. We'll map that and revert.

**Reply by awaisarshd** [reply]
Perfect!!!

**Reply by aditya4196** [reply]
Also, it would be better to break the roadmap into different module based feature roadmaps. As the plugin management is going to be an independent component.
So independent roadmaps can be executed in parallell.
That was also one of the reasons of creating different databases so the main modules are losely coupled.

**Reply by webcrumbs-community** [reply]
Yes, @aditya4196! Exactly.

I'd love some help to think about it. I'll make a first suggestion soon and we build from there

**Reply by awaisarshd** [reply]
Hi @webcrumbs-community, @aditya4196 how are things going?

**Reply by webcrumbs-community** [reply]
Hi, @awaisarshd! I'm working on it. Should have something ready today. I'll post here as soon as I have it

**Comment by webcrumbs-community** [comment]
Hey, @awaisarshd and @aditya4196. I made a first attempt to organize the roadmap here: [WebCrumbs's Public Roadmap](https://github.com/users/webcrumbs-community/projects/1/views/3). I'd like to discuss it with you and set more reasonable timeframes. Would you be up for a call this week before Friday?

**Reply by webcrumbs-community** [reply]
@vbrazo your input is also welcome, my friend!

**Reply by awaisarshd** [reply]
Friday will be fine. Any suggestions for the time?

**Reply by aditya4196** [reply]
Friday works for me too but after 5pm Eastern Standard Time. Otherwise weekend im available entire time.

**Reply by aditya4196** [reply]
@awaisarshd @rovoskarl I would like to discuss about the issue assigned to me regarding the server-side module loading.
Let me know what times are you guys available?. I can try to organize my schedule.

**Reply by webcrumbs-community** [reply]
Friday at 5pm EST works? @awaisarshd

**Reply by awaisarshd** [reply]
@webcrumbs-community and @aditya4196, for me that will be like 3AM in the morning. 

I would suggest to either move few hours earlier like 1 to 2pm EST or push few hours to like 9 or 10PM EST.

**Reply by aditya4196** [reply]
9pm to 12am EST works for me

**Reply by webcrumbs-community** [reply]
If I got it right:

	•	I am at UTC -3.
	•	Aditya is at EST, which is UTC -5.
	•	Awaisar is at UTC +10.

I can't do 9pm on Friday, but I can Thursday.

	•	For me at UTC -3: 11 pm Thursday.
	•	For Aditya at EST (UTC -5): 9 pm the same day.
	•	For Awaisar at UTC +10: 12 pm (noon) Friday.

Does it work?

**Reply by awaisarshd** [reply]
Actually I am in UTC+5 (Pakistan).

For me all days work on the time mentioned by @aditya4196 9pm to 12am

**Reply by webcrumbs-community** [reply]
Damn GPT, I should have checked. Well, that's better. Less of a difference. Key, I'll wait for @aditya4196 to see if Thursday works.

**Reply by webcrumbs-community** [reply]
I forgot today is Thanksgiving. I'll be online tomorrow at discord from 12pm, so let's see if some time works. But we can split too

**Reply by dew-dr0p** [reply]
I'd have loved to join this call tomorrow to discuss the roadmap, but tomorrow seems really occupied for me. I'll still see if I can make a workaround though to join the meeting. I'm at UTC+1 so that would be 4pm, so I'll see if I can make it to the meeting.
What channel would it be on discord?

**Reply by webcrumbs-community** [reply]
Hey, @dew-dr0p. You're very welcome! Thanks a lot indeed. Are you free to chat anytime soon? Because we're in such different timezones perhaps it will work better as many 1-on-1s. Text me at our Discord anytime

**Reply by webcrumbs-community** [reply]
Our discord: https://discord.gg/ymVgdPe6rK

**Reply by dew-dr0p** [reply]
I'm free to chat, I'd message you on discord so we could talk more

**Comment by webcrumbs-community** [comment]
Luckily GitHub has great and free features for Roadmaps. Please check the tabs at this link: https://github.com/users/webcrumbs-community/projects/1/views/3 and drop a comment here or there (discord) if you have any suggestions!

**Reply by AJBrownson** [reply]
Nice one @webcrumbs-community 
Y'all are doing great work
So sorry for my absence these days
I've been really occupied

**Reply by webcrumbs-community** [reply]
Thanks, @AJBrownson. Great that you're back!
